client/packages/system/src/Immunisation/VaccineCourseView/VaccineCourseView.tsx

```diff
@@ -104,6 +105,9 @@ export const VaccineCourseView: FC = () => {
     query: { data, isLoading },
     isDirty,
   } = useVaccineCourse(id);
+  const {
+    query: { data: programData },
+  } = useImmunisationProgram(t, data?.programId ?? '');
```

Comment:
wondering if it might be nicer to return the program name on the vaccine course node??

Reply:
Probably would be cleaner, I'm fine with this though...